modules/default/newsfeed/newsfeed.js

```diff
@@ -176,19 +180,23 @@ Module.register("newsfeed", {
 			});
 		}
 	},
-
+	getFeedProperty(feed, property) {
+		return feed[property] || defaultValues[property];
+	},
 	/**
 	 * Generate an ordered list of items for this configured module.
 	 * @param {object} feeds An object with feeds returned by the node helper.
 	 */
 	generateFeed (feeds) {
 		let newsItems = [];
-		for (let feed in feeds) {
-			const feedItems = feeds[feed];
-			if (this.subscribedToFeed(feed)) {
+		for (let i = 0; i < feeds.length; i++) {
+			const feedItems = feeds[i];
+			if (this.subscribedToFeed(feedItems)) {
 				for (let item of feedItems) {
-					item.sourceTitle = this.titleForFeed(feed);
-					if (!(this.config.ignoreOldItems && Date.now() - new Date(item.pubdate) > this.config.ignoreOlderThan)) {
+					let ignoreOldItems = getFeedProperty(feedItems, 'ignoreOldItems');
```

Comment:
I changed how the the variables are accessed when trying to display the feed by using the getFeedProperty()